addons/colorblind/bold-links.css

```
@@ -5,13 +5,15 @@ a:hover,
 a:active,
 a.black:hover,
 a.black:active,
+a.reply span,
```

We can probably apply it to `a` instead of `span`, that's what Scratch 2.0 → 3.0 does while you're hovering over the reply button.
```suggestion
a.reply,
```

---

It looks better to have it on the span only, but sure, for consistency that makes sense.